metaflow/plugins/aws/batch/batch.py

```diff
@@ -175,8 +145,17 @@ def create_job(
             .job_name(job_name) \
             .job_queue(queue) \
             .command(
-                self._command(self.environment, code_package_url,
-                              step_name, [step_cli], task_spec)) \
+                build_task_command(
+                    self.environment,
+                    code_package_url,
+                    step_name=step_name,
+                    run_id=run_id,
+                    task_id=task_id,
+                    attempt_str=attempt_str,
```

Comment:
this needs to be `attempt=attempt` otherwise the batch integration throws an error.